Client/Frontend/Library/LibraryPanels.swift

```diff
@@ -94,7 +96,8 @@ class LibraryPanels: FeatureFlagsProtocol {
             tabManager: tabManager,
             imageName: "Bookmarks",
             accessibilityLabel: .LibraryPanelBookmarksAccessibilityLabel,
-            accessibilityIdentifier: "LibraryPanels.Bookmarks"),
+            accessibilityIdentifier: "LibraryPanels.Bookmarks",
```

Comment:
nit: imageName should be in ImageIdentifiers, accessibilityIdentifier should be in AccessibilityIdentifier.swift